state-sync/src/request_manager.rs

```diff
@@ -338,14 +340,9 @@ impl RequestManager {
             prev_request.last_request_time = now;
             prev_request.clone()
         } else {
-            self.requests.insert(
-                version,
-                ChunkRequestInfo::new(version, peers, self.multicast_level),
-            );
-            self.requests
-                .get(&version)
-                .expect("missing chunk request that was just added")
-                .clone()
+            let chunk_request_info = ChunkRequestInfo::new(version, peers, self.multicast_level);
+            self.requests.insert(version, chunk_request_info.clone());
+            chunk_request_info
```

Comment:
Lol, that didn't make much sense before.  Let's create something then pull it out of the map :/